### DISCUSSION TITLE
Selection inside `overflow: scroll` container

### DISCUSSION THREAD

**Discussion by mlajtos** [simonwep/viselect#139] [Q&A]
Hello,

I am struggling to get selection working inside a `overflow: scroll` container.

Relevant code: https://codesandbox.io/s/viselect-react-forked-50x7n?file=/src/App.tsx:899-1034
Live demo: https://50x7n.sse.codesandbox.io/

Here is what I get:
https://user-images.githubusercontent.com/1047512/137693145-85b984f1-de76-48b4-b9d4-4f23b9e7598a.mov

What I would like to achieve is that position of the selection area would be fixed inside the scrolling container and auto scroll would also work.

I tried changing position of the selection area parent (`this._area`) from `fixed`, to `static`, `absolute`, and `relative`, but the result was always broken.

Is what I am trying to achieve even possible with this library?

**Comment by simonwep** [comment]
> I tried changing position of the selection area parent (this._area) from fixed, to static, absolute, and relative, but the result was always broken.

Shouldn't be necessary. What kind of browser are you using?

**Reply by mlajtos** [reply]
Safari

**Reply by simonwep** [reply]
Alright, thanks - just wanted to verify that #143 is a duplicate... I'll take a look into it :)
